modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioNextClientCodegen.java

```diff
@@ -182,16 +183,16 @@ private void configureSerializationLibraryBuiltValue(String srcFolder) {
         typeMapping.put("List", "BuiltList");
         typeMapping.put("set", "BuiltSet");
         typeMapping.put("map", "BuiltMap");
-        typeMapping.put("file", "Uint8List");
-        typeMapping.put("binary", "Uint8List");
+        typeMapping.put("file", "MultipartFile");
```

Comment:
That broke file type that are not using multipart to upload file no ??

Reply:
I am not sure but I don't think so. I only see 2 cases for this to be used, either as body type or as multipart form parameter. It just doesn't make sense to have a field in a model that has this type.
I compared this to kotlin/java generators and they use `java.io.File` in all the same places.
A `MultipartFile` can be instantiated from a file or from a byte array, so everyone should be able to adapt.

Reply:
I agree it doesn't make sense, but openapi allow it and it's supported right now with this, if you change that and someone is using it, it will break for them

Reply:
Well, anything that is not a `multipart` request was not supported until now, because there is no serializer for `Uint8List`. And since this is the `dart-dio-next` generator which is marked experimental, breaking changes are to be expected.

Reply:
`dart-dio-next ` is the only generator for dio and null safety, not sure it can be called experimental as we don't have other alternatives ^^
`Uint8List ` doesn't need serializer as it's a primitive type. At least from what I know

Reply:
> `Uint8List ` doesn't need serializer as it's a primitive type. At least from what I know

Yes, you are correct, not sure it makes sense to serialize and int list in JSOn tho.

I will try to add a serializer for `MultipartFile` which serializes it back to a `Uint8List`.

Reply:
I think Uint8List is completely pointless here and if somebody needs it - it is a super easy fix, just create the MultipartFile from Uint8List at the invocation level

Reply:
Yea, but I now changed it to use `MultipartFile` only for body or mulitpart parameters, and I added a test.